apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -62,52 +62,67 @@ contract DisputableVoting is IForwarderWithContext, DisputableAragonApp {
 
     enum VoteStatus {
         Active,                         // An ongoing vote
-        Paused,                         // A vote that is being challenged
-        Cancelled,                      // A vote that has been cancelled since it was refused after a challenge or dispute
+        Paused,                         // A vote that is paused due to it having an open challenge or dispute
+        Cancelled,                      // A vote that has been explicitly cancelled due to a challenge or dispute
         Executed                        // A vote that has been executed
     }
 
     struct Setting {
+        // Required voter support % (yes power / voted power) for a vote to pass
+        // Expressed as a percentage of 10^18; eg. 10^16 = 1%, 10^18 = 100%
         uint64 supportRequiredPct;
+
+        // Required voter quorum % (yes power / total power) for a vote to pass
+        // Expressed as a percentage of 10^18; eg. 10^16 = 1%, 10^18 = 100%
+        // Must be <= supportRequiredPct to avoid votes being impossible to pass
         uint64 minAcceptQuorumPct;
-        uint64 executionDelay;
+
+        // Duration before the end of a vote to stop allowing representatives to vote and for principals to override their representative's vote
+        // Must be <= voteTime
         uint64 overruleWindow;
+
+        // Duration before the end of a vote to detect non-quiet endings
+        // Must be <= voteTime
         uint64 quietEndingPeriod;
+
+        // Duration to extend a vote in case of non-quiet ending
         uint64 quietEndingExtension;
+
+        // Duration to wait before a passed vote can be executed
+        uint64 executionDelay;
```

Comment:
I thought it'd be useful to describe these settings a bit more, although we will likely want to revisit these descriptions once we re-think these "windows" to be based on the start time rather than end time.